modules/policy-criteria.adoc

```diff
@@ -865,12 +865,18 @@ NOT, +
 | *Runtime* ONLY - Kubernetes Events
 
 | Kubernetes Resource
-| The name of the accessed Kubernetes resource, such as `configmaps` or `secrets`.
+| Type of the accessed Kubernetes resource.
 | Kubernetes Resource
 | One of: +
 
-SECRETS +
-CONFIGMAPS +
+Config maps +
```

Comment:
Just checking if this is correct because this one has a space in it, but all others don't.

(Also not sure if this is an official list pulled from somewhere that uses these actual values, or if they should either be listed as their general name ("Cluster roles") or API object formatting (`ClusterRole`) )?

Reply:
They are from a drop-down menu in the GUI, so the space is in there. (Van posted a screenshot above.) Normally I would bold GUI items but none of them in this table are bolded, so I think I should be consistent with the rest of the table.